app/controllers/contacts_controller.rb

```diff
@@ -99,6 +101,52 @@ def contact
     end
   end
 
+  def edit_profile_action
+    formValues = JSON.parse(params.require(:formValues), symbolize_names: true)
+    edited_profile_details = formValues[:editedProfileDetails]
+    edit_profile_reason = formValues[:editProfileReason]
+    contact_email = formValues[:contactEmail]
+    attachment = params[:attachment]
+    wca_id = formValues[:wcaId]
+    if current_user.present?
+      profile_to_edit = {
+        name: current_user.person.name,
+        country_iso2: current_user.person.country_iso2,
+        gender: current_user.person.gender,
+        dob: current_user.person.dob,
+      }
+    else
+      person = Person.find_by(wca_id: wca_id)
+      profile_to_edit = {
+        name: person.name,
+        country_iso2: person.country_iso2,
+        gender: person.gender,
+        # DOB should not be visible to unauthenticated users.
+      }
+    end
+    changes_requested = PERSON_FIELDS_EDIT_REQUESTABLE
+                        .reject { |field| profile_to_edit[field].to_s == edited_profile_details[field].to_s }
+                        .map { |field|
+                          {
+                            field: field.to_s.humanize,
+                            from: (new_profile_data_key_to_value(profile_to_edit[field], field.to_s) || "Unknown").to_s,
+                            to: new_profile_data_key_to_value(edited_profile_details[field], field.to_s),
+                          }
+                        }
+
+    maybe_send_contact_email(
```

Comment:
What is the `maybe` part about? In which cases do we not send an email?

Reply:
This method was named so from very long time - I think even before I started working on contact form. The `may` is not relevant much because it fails only when there is a server error or validation error, but since it's used in many places I didn't change it. But I can change all of them all-together in any of the future PRs if needed.

Reply:
Oh, I didn't realize that. Thought you came up with this method.

But then nevermind, never touch a running system :sweat_smile: